Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.8-buster
+
+WORKDIR /app
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONIOENCODING=utf-8
+ENV PYTHONPATH=/app
+
+RUN pip3 install --upgrade pip && rm -r ~/.cache/pip/
+
+COPY ./requirements /app/requirements
+RUN pip3 install \
+    -r ./requirements/requirements_base.txt \
+    -r ./requirements/requirements_test.txt \
+    -r ./requirements/requirements_prometheus.txt \
+    && rm -r ~/.cache/pip/
```

Comment:
Why not use make commands?

Reply:
Because this would require copying the Makefile to the container with the requirements files.
Also, I was looking at the code and didn't find any reference to the tests requirements on the Makefiles.

Reply:
Take a look in our [Contributing Guide](https://github.com/olxbr/BarterDude/blob/master/CONTRIBUTING.md). Foi this issue, `make setup` should install all requirements needed

Reply:
Sorry about that. Missed that line when reading the Makefile.

Reply:
Changing that would make every change in code requiring rebuilding and downloading all the packages, since `make setup` calls `install`, which install everything in editable mode.

If you are ok if that I can change.